Final Sharks and Minnows

ptownazkikr opened the pull request:
I added a particle emitter to the sharks and gave both of them bodies.

ptownazkikr committed:
Merge branch 'master' into Sharks&Minnows2

ptownazkikr committed:
Minor fixes

Repulsion is more prominent. More sharks

ptownazkikr committed:
Merge branch 'master' into Sharks&Minnows2

Commit:
Added primitive Models and Blood

Made the sharks and minnows bodies out of primitive 3D shapes, gave them both simple materials to set their albedos, and added a particle emitter to the shark that plays when it eats a minnow.

SweaterWeather commented:
I have to deny your pull request for now on account of lack of proper commenting.  please add comments to your spawners and attractors/repulsors.  also, please finish commenting the minnow and shark creatures themselves, each is only about half commented.  I also noticed that the minnows and sharks seem to be able to turn on a dime, almost instantly switching from one direction to the other.  their movements would look more convincing if they had a slower turn speed

your fish are also very complicated, each of them is actually made out of several nested game objects.  please combine the cylinders into a single mesh instead, we can reduce our total number of game objects to 1/3 or so if these game objects are combined into a single mesh

Commit:
Updates

Comments were made, turn speed was slowed, and the meshes were combined.

Commit:
Merge branch 'master' into Sharks&Minnows2

Commit:
Removed MeshStitcher

Removed MeshStitcher on account of it not being directly needed and also not being up to repo standards (no comments)

Commit:
Added missing comments
